tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/observability/metrics/MetricsTestCase.java

```diff
@@ -48,7 +48,7 @@
 /**
  * Integration test for observability of metrics.
  */
-@Test(groups = "metrics-test")
+@Test(groups = {"metrics-test", "brokenOnTableRemove"}, enabled = false)
```

Comment:
@niveathika, any reason to change this?

Reply:
Because these metrics are calculated for a bal file using JDBC std lib